02-peer-protocol.md

```diff
@@ -926,6 +926,25 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
    * [`sha256`:`payment_hash`]
    * [`u32`:`cltv_expiry`]
    * [`1366*byte`:`onion_routing_packet`]
+   * [`update_add_htlc_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `update_add_htlc_tlvs`
+2. types:
+    1. type: 0 (`hold_htlc`)
+    2. data:
+        * [`32*bytes`:`payment_release_secret`]
```

Comment:
Per our work in LDK (cc https://github.com/lightningdevkit/rust-lightning/pull/3140/commits/6e27aecb408b79d1c5d3383a3e3cc331691a95c7), this needs an update to remove the `payment_release_secret` and authenticate these messages via the blinded path only.